.eslintrc.js

```diff
@@ -1,6 +1,6 @@
 module.exports = {
   root: true,
-  extends: "babel",
+  parser: "lightscript-eslint",
```

Comment:
I get why we need to get rid of `extends: "babel"`, I'm confused why we don't have an `extends: "lightscript"` and why we need a `parser: "lightscript-eslint"` if the parser wasn't needed before...